### PR TITLE
cloud: Add password policy

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/manage-user-access.md [hunk 1]
```diff
@@ -25,6 +25,26 @@ After you have signed into TiDB Cloud, if you need to sign out, perform the foll
...
+
+1. Click the account name in the upper-right corner of the window.
+
+2. Click **Account**.
+
+3. Click the **Change Password** tab.
+
+4. Click **Change Password**, and then check your mail box for the link to reset the password.
+
+If your password is not changed within 90 days, you will receive a prompt to reset your password when you log in to TiDB Cloud. In this case, you need to follow the prompt to reset the password.
```
Comment (qiancai) [inline tidb-cloud/manage-user-access.md:46]:
```suggestion
If your password is not changed within 90 days, you will get a prompt to reset your password when you log in to TiDB Cloud. In this case, you need to follow the prompt to reset the password.
```

File: tidb-cloud/manage-user-access.md [hunk 2]
```diff
@@ -25,6 +25,26 @@ After you have signed into TiDB Cloud, if you need to sign out, perform the foll
 
 2. Click **Logout**.
 
+## Manage user passwords
+
+> **Note:**
+>
+> The content in this section is only applicable to TiDB Cloud registration with email and password. If you sign up for TiDB Cloud with Google or GitHub, your password is managed by Google or GitHub and cannot be changed using TiDB Cloud console.
```
Comment (qiancai) [inline tidb-cloud/manage-user-access.md:32]:
```suggestion
> The content in this section is only applicable to TiDB Cloud registration with email and password. If you sign up for TiDB Cloud with Google or GitHub, your password is managed by Google or GitHub and cannot be changed using the TiDB Cloud console.
```

**Review by lulukelu — APPROVED** [review]

**Review by ljun0712 — APPROVED** [review]

File: tidb-cloud/manage-user-access.md [hunk 3]
```diff
@@ -25,6 +25,26 @@ After you have signed into TiDB Cloud, if you need to sign out, perform the foll
 
 2. Click **Logout**.
 
+## Manage user passwords
+
+> **Note:**
+>
+> The content in this section is only applicable to TiDB Cloud registration with email and password. If you sign up for TiDB Cloud with Google or GitHub, your password is managed by Google or GitHub and cannot be changed using the TiDB Cloud console.
```
Comment (hfxsd) [inline tidb-cloud/manage-user-access.md:32]:
```suggestion
> The content in this section is only applicable to TiDB Cloud registration with email and password. If you sign up for TiDB Cloud with Google or GitHub, your password is managed by Google or GitHub and you cannot change it using the TiDB Cloud console.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Pull request merged**